java/org/apache/catalina/mbeans/MBeanFactory.java

```diff
@@ -490,6 +491,7 @@ public String createStandardContext(String parent,
     public String createStandardHost(String parent, String name,
                                      String appBase,
                                      boolean autoDeploy,
+                                     boolean parallelAnnotationScanning,
```

Comment:
I disagree with this API change. IMO this new feature does not justify it.

Reply:
Thanks @rmaucher for the comment. 
I have little experience working on Tomcat so if you can you explain how this becomes irrelevant that would be great. Also, do we not want the API's to be able to pass this flag when the optimizations are quite significant?

Reply:
This is not enough to justify changing this API. You can create the host, then set the flag just after. Good enough.

Reply:
Alright. I have removed these changes. 
Resolved with the latest commit. 
Thanks 👍